src/gui/uithememanager.h

```diff
@@ -62,6 +62,7 @@ class UIThemeManager : public QObject
     QIcon getIcon(const QString &iconId, const QString &fallback = {}) const;
     QIcon getFlagIcon(const QString &countryIsoCode) const;
 
+    bool isDarkTheme() const;
```

Comment:
Isn't it about "darkness" of system theme?

Reply:
Yes, I don't really get why you asked? Or do you mean move it to `Utils::Gui`?

Reply:
> I don't really get why you asked?

I don't like it as part of UIThemeManager interface. It creates a false impression that qBittorrent uses a dark theme currently.


> Or do you mean move it to `Utils::Gui`?

At least that.

Reply:
> > Or do you mean move it to `Utils::Gui`?
> 
> At least that.

Done.

>It creates a false impression that qBittorrent uses a dark theme currently.

Can't relate. The function only tells whether qbt current theme is dark or not.

Reply:
> The function only tells whether qbt current theme is dark or not.

I'm sorry, you're confusing me after all. What is it really about? About the system theme or about the qBittorrent theme?

Reply:
>About the system theme or about the qBittorrent theme?

qbt current theme.

Reply:
> qbt current theme, the one shown on your qbt instance.

Then I doubt the usefulness of this function. What could be interested in it? It does not seem that in a well-designed application, any code outside the theme management subsystem could need this information.

Reply:
>What could be interested in it? 

Custom colors not defined by any standards (such as #17714) can take the dark/light theme into consideration and provide different color palette.

Reply:
Shouldn't all these colors come from the theme?

Reply:
>Shouldn't all these colors come from the theme?

OS theme don't provide those.

Reply:
>I strongly don't believe a single color palette will be good enough for both dark/light themes

Agree.

Reply:
As for #17714, if someone intends to use custom colors in Log widget, then they should add support for custom themes as well (like in transfer list widget).

Reply:
>As for https://github.com/qbittorrent/qBittorrent/pull/17714, if someone intends to use custom colors in Log widget, then they should add support for custom themes as well (like in transfer list widget).

It seems log view already has proper support for qbt themes. #17714 is just tweaking the color for OS native theme. Also I plan to look into the colors myself later.

Reply:
> It seems log view already has proper support for qbt themes. #17714 is just tweaking the color for OS native theme. Also I plan to look into the colors myself later.

Ok. I misread that it just changes default values while custom ones are still loaded from theme file.